src/components/Checkbox.js

```diff
@@ -17,13 +19,15 @@
 /**
  * @typedef LabeledCheckboxBaseProps
  * @prop {import('preact').ComponentChildren} children - Label text or elements
- * @prop {'before'|'after'} [position] - Position the label before or after the checkbox. Defaults to 'before'.
  *
  * @typedef {Omit<CheckboxProps, 'children'> & LabeledCheckboxBaseProps} LabeledCheckboxProps
  */
 
 /**
- * A checkbox input
+ * A checkbox input.
+ *
+ * A checkbox component is a combination of an <input> element and a sibling
+ * <svg> element that is used for the visual appearance of the checkbox.
```

Comment:
This isn't quite accurate and is stale from a previous implementation cycle; this should read `<SvgIcon>` instead of `<svg> element`